src/System.Net.Http/src/System/Net/Http/Unix/CurlResponseHeaderReader.cs

```diff
@@ -111,6 +119,18 @@ private static bool ValidHeaderNameChar(byte c)
             return c > ' ' && invalidChars.IndexOf((char)c) < 0;
         }
 
+        internal static bool IsWhiteSpaceLatin1(byte c)
+        {
```

Comment:
You should add a comment about the space character (' ') as well since you listed all the others.